src/AppBundle/Entity/Tag.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Tag
+ *
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\TagRepository")
+ *
+ * Defines the properties of the User entity to represent the application users.
+ * See http://symfony.com/doc/current/book/doctrine.html#creating-an-entity-class
+ *
+ * Tip: if you have an existing database, you can generate these entity class automatically.
+ * See http://symfony.com/doc/current/cookbook/doctrine/reverse_engineering.html
+ *
+ * @author Rasanga Perera <rasangaperera@gmail.com>
+ */
+class Tag
+{
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="tag_name", type="string", length=255)
```

Comment:
it is not a reserved keyword in any platform supported by Doctrine (see https://github.com/doctrine/dbal/tree/master/lib/Doctrine/DBAL/Platforms/Keywords for the reserved keywords on each platform).
And the demo runs on MySQL anyway, so only mysql reserved keywords are important.